src/intro_movie_scene_2.c

```diff
@@ -0,0 +1,457 @@
+    }
+    IntroMovie_DestroySpriteResourceManagers(data);
+}
+
+void IntroMovie_Scene2_CreateSprites(IntroMovieOvyData *data, IntroMovieScene2Data *sceneData) {
+    SpriteResourcesHeader spriteHeader;
+    SpriteTemplate spriteTemplate;
+    u8 anims[10] = {
+        0, 1, 2, 3, 0, 1, 2, 3, 0, 1,
+    };
+
+    IntroMovie_BuildSpriteResourcesHeaderAndTemplate(0, data, 0, NNS_G2D_VRAM_TYPE_MAX, &spriteTemplate, &spriteHeader);
+    spriteTemplate.position.x = 128 * FX32_ONE;
+    spriteTemplate.position.y = 192 * FX32_ONE;
+
+    for (u8 i = 0; i < 10; ++i) {
+        sceneData->flowerSprites[i] = CreateSprite(&spriteTemplate);
+        Set2dSpriteAnimActiveFlag(sceneData->flowerSprites[i], FALSE);
+        Set2dSpriteVisibleFlag(sceneData->flowerSprites[i], FALSE);
+        Set2dSpriteAnimSeqNo(sceneData->flowerSprites[i], anims[i]);
+    }
+
+    IntroMovie_BuildSpriteResourcesHeaderAndTemplate(1, data, 1, NNS_G2D_VRAM_TYPE_2DMAIN, &spriteTemplate, &spriteHeader);
+    spriteTemplate.position.x = 64 * FX32_ONE;
+    spriteTemplate.position.y = -96 * FX32_ONE;
+    sceneData->ethanSprite0 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->ethanSprite0, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->ethanSprite0, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->ethanSprite0, 0);
+    sceneData->ethanSprite1 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->ethanSprite1, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->ethanSprite1, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->ethanSprite1, 1);
+    sceneData->ethanSprite2 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->ethanSprite2, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->ethanSprite2, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->ethanSprite2, 2);
+
+    IntroMovie_BuildSpriteResourcesHeaderAndTemplate(1, data, 1, NNS_G2D_VRAM_TYPE_2DMAIN, &spriteTemplate, &spriteHeader);
+    spriteTemplate.position.x = 320 * FX32_ONE;
+    spriteTemplate.position.y = -96 * FX32_ONE;
+    sceneData->lyraSprite0 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->lyraSprite0, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->lyraSprite0, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->lyraSprite0, 3);
+    sceneData->lyraSprite1 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->lyraSprite1, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->lyraSprite1, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->lyraSprite1, 4);
+    sceneData->lyraSprite2 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->lyraSprite2, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->lyraSprite2, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->lyraSprite2, 5);
+}
+
+void IntroMovie_Scene2_StartSpritePanEffect(IntroMovieScene2Data *sceneData, int whichSprites, int xSpeed, int ySpeed, int duration) {
+    IntroMovieScene2SpritePanController *spritePanEffect;
+
+    if (whichSprites == 0) {
```

Comment:
ins't this the gender const?

Reply:
looks like it, but we don't have a deefined gender const pair that properly fits this use case